src/mol-plugin-ui/skin/base/components/viewport.scss

```
@@ -173,15 +173,13 @@
     background: rgba(red($default-background), green($default-background), blue($default-background), 0.5);
 
     position: absolute;
+    height: 50vh; // 50% of the viewport height
     left: 0;
     top: $control-spacing + $row-height;
     padding: (0.66 * $control-spacing) $control-spacing;
 
-    max-height: 7 * $row-height;
-    overflow: hidden;
-    overflow-y: auto;
-
-    max-width: 400px;
+    resize: both; /* Allows resizing in both directions */
+    overflow: scroll; /* Adjust as needed */
```

```suggestion
    overflow: auto; /* Adjust as needed */
```

The `scroll` will always show the scrollbars in some viewers, I am not quite sure this is desirable.